src/components/Signup.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { connect } from "react-redux";
+import auth from "../modules/auth";
+
+const Signup = props => {
+
+  const onSignup = event => {
+    event.preventDefault();
+    auth
+      .signUp({
+        email: event.target.email.value, 
+        password: event.target.password.value
+      }, "http://localhost:3000")
```

Comment:
Don't hardcode to localhost

Reply:
And maybe fix the hardcoding in the auth-module while you're at it :)

Reply:
Added to adress chore